website/blog/2023-01-12-version-071.md

```diff
@@ -0,0 +1,246 @@
+
+- [`objectFit`](https://reactnative.dev/docs/next/image#objectfit)
+- `pointerEvents`
+- `userSelect`
+- [`verticalAlign`](https://reactnative.dev/docs/next/text-style-props#verticalalign-android)
+
+See this [_issue_](https://github.com/facebook/react-native/issues/34425) for more details.
+
+#### Events
+
+In this release we also added an opt-in implementation of [_PointerEvents_](https://w3c.github.io/pointerevents)
+
+Once enabled, the following handlers on `View` will support hover:
+
+- `onPointerOver`, `onPointerOut`
+- `onPointerEnter`, `onPointerLeave`
+
+These events are also implemented in `Pressability` for new opt-in support for hover.
+
+To enable these features, set the following flags to true:
+
+```
+import ReactNativeFeatureFlags from "react-native/Libraries/ReactNative/ReactNativeFeatureFlags";
+
+// enable the JS-side of the w3c PointerEvent implementation
+ReactNativeFeatureFlags.shouldEmitW3CPointerEvents = () => true;
+
+// enable hover events in Pressibility to be backed by the PointerEvent implementation.
+// shouldEmitW3CPointerEvents should also be true
+ReactNativeFeatureFlags.shouldPressibilityUseW3CPointerEventsForHover = () =>
+  true;
+
+```
+
+:::note
+
+You’ll also need to enable React feature flags on your [_Android_](https://reactnative.dev/blog/2022/12/13/pointer-events-in-react-native#android-specific) and [_iOS_](https://reactnative.dev/blog/2022/12/13/pointer-events-in-react-native#ios-specific) native setup.
+
+:::
+
+Check out our dedicated [_PointerEvents post_](https://reactnative.dev/blog/2022/12/13/pointer-events-in-react-native) to learn more.
+
+## Restoring PropTypes
+
+React Native’s prop types, such as `ViewPropTypes` and `Text.propTypes`, were deprecated in 0.66 and accessing them would output deprecation warnings. When they were removed in 0.68, many developers began experiencing errors when upgrading to the latest version of React Native.
+
+After some investigation, we realized that a couple issues prevented the community from taking action on the deprecation warnings. First, the deprecation warning was not always actionable which caused people to ignore them ([issue one](https://facebook/react-native#34650), [issue two](https://github.com/react-native-community/cli#1699)). Second, the deprecation warnings were being [incorrectly filtered](https://facebook/react-native@fa2842d) by `LogBox.ignoreLogs`. Both of these have now been fixed, but we want to give people more time to upgrade the deprecated call sites.
+
+So in this release we are adding back React Native’s prop types so that it is easier for people to upgrade and migrate their code to avoid using them. The `deprecated-react-native-prop-types` package has also been updated for all of the props in 0.71. In the future, we plan to proceed with the deprecation and remove prop types once again. We expect that when we revisit the removal, the community will experience significantly less issues.
```

Comment:
prop types -> propTypes ?

Reply:
```suggestion
So in this release we are adding back React Native’s propTypes so that it is easier for people to upgrade and migrate their code to avoid using them. The `deprecated-react-native-prop-types` package has also been updated for all of the props in 0.71. In the future, we plan to proceed with the deprecation and remove prop types once again. We expect that when we revisit the removal, the community will experience significantly less issues.
```